src/script/components/MessagesList/Message/VerificationMessage.tsx

```diff
@@ -102,7 +102,7 @@ const VerificationMessage: React.FC<VerificationMessageProps> = ({message}) => {
         )}
         {isTypeNewMember && (
           <>
-            <span className="ellipsis">{t('conversationDeviceNewPeopleJoined')}</span>
```

Comment:
Is this class still used somewhere? If not can we remove it?

Reply:
It's still used in quite a lot of places